validation/views.py

```diff
@@ -381,6 +381,64 @@ def make_absolute_uri_for_speaker(code: str) -> str:
     return add_cors_headers(response)
 
 
+def bulk_search_recordings(request):
+    """
+    API endpoint to retrieve EXACT wordforms and return the URLs and metadata for the recordings.
+    Example: /api/bulk_search?q=mistik&q=minahik&q=waskay&q=mîtos&q=mistikow&q=mêstan
+    """
+
+    def make_absolute_uri_for_recording(rec: Recording) -> str:
+        uri = rec.compressed_audio.url
+        if uri.startswith("/"):
+            # It's a relative URI: build an absolute URI:
+            return request.build_absolute_uri(uri)
+
+        # It's an absolute URI already:
+        assert uri.startswith("http")
```

Comment:
I hope it’s ok to be a little bit nitpicky here, but in case some random filename like `http_stuff.txt` starts with `http` someday:

```suggestion
        assert uri.startswith("http://") or uri.startswith("https://")
```